### ISSUE TITLE
FTDNN trainning example- Loss function 

### ISSUE THREAD

**Issue by sfxsalem** [cvqluu/Factorized-TDNN#5]
Hello, 
I am trying to include the FTDNN model that you wrote in the framework pytorch-kaldi. I managed to implement the model ut I have some issues when training which I suspect are related to the input and output I am using. 
Could you possibly provide the code that you used for the demonstration of FTDNN or what king of Loss function should be used at the end ? 
Thank you.

**Comment by cvqluu** [comment]
[pasted and edited from a [previous issue](https://github.com/cvqluu/Factorized-TDNN/issues/3), as it is mostly applicable here]

I'm just leaving this repo up as a very simple snippet for the models alone.

If you'd like more info on the preprocessing and the whole pipeline, check out https://github.com/cvqluu/dropclass_speaker which has a bit more information on how to run this completely, and it includes information on data processing.

Specifically you would make a [config file](https://github.com/cvqluu/dropclass_speaker/blob/master/configs/example_speaker.cfg) with the `model = FTDNN` set. The loss function I used for the baseline was `loss_type = adm` (I think), which is an additive margin softmax loss.

You can ignore all the other features in the config file which are specific to the paper the repo is based around.

If you have any further questions there feel free to bring them up on that repo.

Hope that helps,
Chau
